features/canvas-2d-context-alpha.yml

```diff
@@ -0,0 +1,6 @@
+name: Opaque 2D canvas
```

Comment:
Hmm, I accepted this, but am now having second thoughts, and want to go back to `Alpha` or alternatively I could be OK with `2d canvas opacity`. Every canvas is opaque unless you use this parameter. A quick web search returns better results for either Alpha or opacity.

Reply:
To give you a sense of my thinking here: the default value is `true` (otherwise the browsers introducing the option would suddenly break a bunch of existing canvases), so the _feature_ is opting into a higher-performance opaque canvas, even though the API is (ostensibly) a neutral control over alpha/opacity.

That said, I am convinced by the search results yielding better for opacity, so "2D canvas opacity" would be OK with me.

Reply:
done!